k8s-agent-debug-cluster-config/launchSettings.template.json

```diff
@@ -0,0 +1,29 @@
+{
+  "profiles": {
+    "Octopus Kubernetes Agent": {
+      "commandName": "Project",
+      "commandLineArgs": "agent --instance Tentacle --noninteractive",
+      "framework": "net8.0",
+      "environmentVariables": {
+        "ACCEPT_EULA": "Y",
+        "OCTOPUS__K8STENTACLE__NAMESPACE": "octopus-agent-<agent-name>",
```

Comment:
Ah yes, didn't think about this. I was considering updating the script to query the cluster and find the agent by finding a helm release that has a matching name or something. But I thought initially this would work. I'm thinking maybe lets get this first version in and we can iterate on it 👍